src/components/AddBook.js

```diff
@@ -18,25 +18,29 @@ const AddBook = () => {
     console.log('Book; submitted ✅');
   };
   return (
-    <>
-      <form onSubmit={handleSubmit}>
+    <div className='container mt-5 pb-5'>
+      <div class="Line"></div>
+    <h5 className='text-muted fw-bold'>Add New Book</h5>
+      <form onSubmit={handleSubmit} className="d-flex justify-content-between ">
         <input
+        className='w-50 p-2 text-muted'
           id="bookName"
           type="text"
           name="title"
           required
           placeholder="Book Name"
         />
         <input
+        className='w-25 p-2 text-muted'
           id="author"
           type="text"
           name="author"
           required
           placeholder="Author"
         />
-        <button type="submit" >Add Book</button>
+        <button type="submit" className='btn btn-sm btn-primary' >Add Book</button>
       </form>
-    </>
```

Comment:
- [ ] You should stick to the design as much as possible that is color, font, and spacing. please adjust the spacing between the input field and use the same color in the Figma design
### Your design
![yourrrrrrrrrrrr](https://user-images.githubusercontent.com/81727988/202830787-4a20305b-a6fb-4d9f-84e4-3a8a5cd5d008.png)
### Figma design
![figmaaaaaaaaaaa](https://user-images.githubusercontent.com/81727988/202830797-9ee7248f-22ee-45f9-9781-08b5ab8db4db.png)